products/render-game.js

```diff
@@ -14,17 +14,33 @@ function renderGame(game) {
     img.alt = game.name + ' image';
     li.appendChild(img);
 
+    // create dropdown
+    const select = document.createElement('select');
+    select.id = 'select';
+
+    // to get the value it is select.value
+    li.appendChild(select);
+
+    // loop through and add 100 options to the dropdown
+    for (let i = 1; i < 100; i++) {
```

Comment:
You've actually added 99 options :) 
If you use <= 100 you'll get the last one.